cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Probably just remove this file given that its entirely commented out? Or are we leaving it here for future potential use?

Reply:
Cypress creates it when initialized, haven't looked at it since. Its useful information if we ever need it 🤷‍♂️